### DISCUSSION TITLE
QED symbol at the end of a proof

### DISCUSSION THREAD

**Discussion by gdietl** [quarto-dev/quarto-cli#5033] [Q&A]
In the proof environment of bookdown, the end has been marked by a square, commonly used as the QED symbol. However, using quarto, the end of the proof is not specified. Is there an easy way to add this symbol to the proof environment?

**Comment by cderv** [comment]
I think we should add it by default yes. We can probably port  what we did in **bookdown**. Thanks ! I'll create an issue

**Comment by ghost** [comment]
Thanks @cderv moved to https://github.com/quarto-dev/quarto-cli/issues/5036
